src/test_main.py

```diff
@@ -70,14 +71,14 @@ def getitem(name):
         imap_connection.fetch.assert_called_once_with("a", "(RFC822)")
         email.message_from_bytes.assert_called_once_with(b"raw")
         tasks_from_email.convert_to_kb_date.assert_has_calls(
-            [mocker.call("rfcdate"), mocker.call("rfcdate", 48)]
+            [call("rfcdate"), call("rfcdate", 48)]
         )
         assert email_message.__getitem__.call_args_list == [
-            mocker.call("Date"),
-            mocker.call("Date"),
-            mocker.call("From"),
-            mocker.call("To"),
-            mocker.call("Subject"),
+            call("Date"),
+            call("Date"),
```

Comment:
why double `call("Date")`?

Reply:
once as from and once as to data or something like that

we should refactor these to be named args at some point for readability.